Item Fixes:

Jedzkie opened the pull request:
Issue #980

Jedzkie committed:
Fixes 'Glorious Jamadhar' (ID: 1282)

  - 'IgnoreDefRate' effect must be 'CriticalAddRace' (kRO)

Reference: http://www.divine-pride.net/database/item/1282

Jedzkie committed:
Fixes 'Ghost Bandana' (ID: 2274)

- Pre-renewal Ghost reduction must be 15% (kRO)
- Renewal Effect must have Ghost property reduction by 15% (kRO)

Reference: http://ro.gnjoy.com/guide/runemidgarts/itemview.asp?itemSeq=2&ID=2274

Jedzkie committed:
Fixes 'Ring of Recovery' (ID: 2936)

- Missing effect for MaxHP + 250, MaxHP +5%, and Heal Power +5%

Reference: http://ro.gnjoy.com/guide/runemidgarts/itemview.asp?itemSeq=2&ID=2936

Jedzkie committed:
Fixes 2 RWC 2012 Rings (ID: 2968 & 2969)

 - Missing effect of Matk +1%

Reference: http://ro.gnjoy.com/guide/runemidgarts/itemview.asp?itemSeq=2&ID=2968
http://ro.gnjoy.com/guide/runemidgarts/itemview.asp?itemSeq=2&ID=2969

Jedzkie committed:
Fixes 'Neko Mimi Kafra' (ID: 5387)

 - Missing status effect when hit. (kRO)
 - Update Trade Restriction. (Renewal)

Reference: http://ro.gnjoy.com/guide/runemidgarts/itemview.asp?itemSeq=2&ID=5387

zackdreaver commented on db/pre-re/item_db.conf:
```diff
@@ -22075,8 +22075,8 @@ item_db: (
-		bonus bAgi,2;
-		bonus2 bSubEle,Ele_Ghost,10;
+		bonus bAgi, 2;
+		bonus2 bSubEle, Ele_Ghost, 10;
```
it's 15% in kRO...
but 10% in fRO aegis

Jedzkie replied:
Even in pre-re? Its 15%?

zackdreaver replied:
pre-renewal kRO is hard to find this day,
I'll use twRO's pre-renewal instead.
And yes, it's 15%
ftp://ropatch.gameflier.com/Patch/2010-05-17itemdata_main.gpf

Jedzkie replied:
Done. Thank you.

Jedzkie committed:
Fixes 'Spiked Scarf' (ID: 5462)

 - Effect should be 'Atk +30' and 'MaxHP -2%'
 - Change from ID to Constant of 'Spike Scarf Box' item script.

Reference: http://ro.gnjoy.com/guide/runemidgarts/itemview.asp?itemSeq=2&ID=5462

Jedzkie committed:
Fixes 'Rainbow Scarf' (ID: 5463)
 - Renewal effect should be 'Matk +30' and 'MaxSP -2%'
 - Change from ID to Constant of 'Rainbow Scarf Box' item.

Reference: http://ro.gnjoy.com/guide/runemidgarts/itemview.asp?itemSeq=2&ID=5463

Jedzkie committed:
Fixes 'Helm of Dragon' (ID: 5467)
 - Wrong item effect for Renewal.
 - Update Trade Restriction.

Reference: http://ro.gnjoy.com/guide/runemidgarts/itemview.asp?itemSeq=2&ID=5467

Jedzkie committed:
Fixes 'Noble Hat' (ID: 5469)

 - Wrong item effect.

Reference: http://ro.gnjoy.com/guide/runemidgarts/itemview.asp?itemSeq=2&ID=5469

Jedzkie committed:
Fixes 'King Tiger Doll Hat' (ID: 5497)

 - Added missing additional damage on Brute monsters.
 - Added Autobonus effect with correct chances.

Reference: http://ro.gnjoy.com/guide/runemidgarts/itemview.asp?itemSeq=2&ID=5497

HerculesWSAPI commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/herculesws/hercules/1170)
<!-- Reviewable:end -->
